### ISSUE TITLE
[Bug] CANNOT LOAD NEWEST EXSPANSION (platform = xbox series x)

### ISSUE THREAD

**Issue by yodathedog** [Teardown-Issue-Tracker-Maintainers/Teardown-Issue-Tracker#580]
### Describe the bug

the newest expansion "folkrace" completely shit itself. i managed to complete the first two cups, and the first two derbys. i unlocked the first off roader, and the second "wagon" after completling the forklift derby. the game becomes LITERALLY unplayble. i cant even open the campaign, because it just fucking crashes. it either crashes, or it never. fucking. loads. its just an infinite loading screen.

### Steps to reproduce the behavior

step 0. make sure your playing on xbox...
step 1. complete first cup.
step 2. complete harvester derby
step 3. purchase second wagon.
step 4. complete second cup.
step 5. complete forklift derby.
step 6. wish you where dead because the GAME CRASHED (when hitting continue), AND NOW IT DOESN'T LOAD THE CAMPAIGN...LIKE... AT ALL.

### Expected behavior

so basically... it should load up the campaign mode without crashing.

### Environment

- Teardown Version = the same day that the folkrace dlc was released.
- platform = xbox series x.

### Additional context

it was the direct result of continuing after completing the forklift derby, and i was playing on an xbox series x. this was about 2 hours after it was released to the public.

**Comment by yodathedog** [comment]
bro was heated.

**Comment by Gregory-Gregory** [comment]
@yodathedog hi!

This issue should be fixed in the upcoming hotfix. At the moment, there is no ETA, but the team hopes to release it as soon as possible.

**Comment by YuLun-bili** [comment]
no response for over a year, closing
